### PR TITLE
fix: step count in LR scheduler

### PR DESCRIPTION
**Pull request by charlesmindee**
This PR fixes the step count for the learning rate scheduler of both tensorflow training scripts.

Any feedback is welcome!

### CODE REVIEW

**Comment by charlesmindee** [comment]
ExponentialDecay for now, since CosineDecay is experimental in 2.4.1

File: references/detection/train_tensorflow.py [hunk 1]
```diff
@@ -148,8 +148,11 @@ def main(args):
 
     # Optimizer
     total_steps = int(args.epochs * len(train_set) / args.batch_size)
```
Comment (fg-mindee) [inline references/detection/train_tensorflow.py:150]: you can remove this line

File: references/detection/train_tensorflow.py [hunk 2]
```diff
@@ -148,8 +148,11 @@ def main(args):
...
-    scheduler = tf.keras.optimizers.schedules.CosineDecay(
-        initial_learning_rate=args.lr, decay_steps=total_steps, alpha=0.0
+    scheduler = tf.keras.optimizers.schedules.ExponentialDecay(
+        args.lr,
+        decay_steps=total_steps,
```
Comment (fg-mindee) [inline references/detection/train_tensorflow.py:153]:
```python
decay_steps=args.epochs * len(train_loader),
```

File: references/recognition/train_tensorflow.py [hunk 3]
```diff
@@ -146,8 +146,11 @@ def main(args):
 
     # Optimizer
     total_steps = int(args.epochs * len(train_set) / args.batch_size)
```
Comment (fg-mindee) [inline references/recognition/train_tensorflow.py:148]: same here

File: references/recognition/train_tensorflow.py [hunk 4]
```diff
@@ -146,8 +146,11 @@ def main(args):
...
-    scheduler = tf.keras.optimizers.schedules.CosineDecay(
-        initial_learning_rate=args.lr, decay_steps=total_steps, alpha=0.0
+    scheduler = tf.keras.optimizers.schedules.ExponentialDecay(
+        args.lr,
+        decay_steps=total_steps,
```
Comment (fg-mindee) [inline references/recognition/train_tensorflow.py:151]: same here

**Review by fg-mindee — CHANGES_REQUESTED** [review]
Thanks! I added an improvement suggestion :ok_hand:

**Review by fg-mindee — APPROVED** [review]
Thanks!
